weave-js/src/components/PagePanelComponents/Home/Browse2/browse2Util.ts

```diff
@@ -7,7 +7,11 @@ export const flattenObjectPreservingWeaveTypes = (obj: {
   [key: string]: any;
 }) => {
   return flattenObject(obj, '', {}, (key, value) => {
-    return typeof value !== 'object' || value == null || value._type == null;
+    return (
```

Comment:
This is a little followup fix from the Image PR i just put in - it is unrelated to refs